apple/internal/codesigning_support.bzl

```diff
@@ -625,7 +623,7 @@ def _post_process_and_sign_archive_action(
       platform_prerequisites: Struct containing information on the platform being targeted.
       process_and_sign_template: A template for a shell script to process and sign as a file.
       provisioning_profile: The provisioning profile file. May be `None`.
-      resolved_codesigningtool: The `struct` from resolve_tools representing the code signing tool.
+      codesigningtool: The files_to_run for the code signing tool.
```

Comment:
Can you move this, and other renames (like line 603 above and 774 below), to keep the arguments alphabetical?

Reply:
I think I fixed all of them, PTAL.